charts/csi-powermax/templates/_helpers.tpl

```diff
@@ -48,3 +48,11 @@ Return the appropriate sidecar images based on k8s version
     {{- end -}}
   {{- end -}}
 {{- end -}}
+
+{{- define "csi-powermax.isStorageCapacitySupported" -}}
+{{- if eq .Values.storageCapacity.enabled true -}}
+  {{- if and (eq .Capabilities.KubeVersion.Major "1") (ge (trimSuffix "+" .Capabilities.KubeVersion.Minor) "25") -}}
```

Comment:
Storage Capacity Tracking feature was GA in K8s 1.24.
The last condition should be "24" instead of "25"

Reply:
Changed the version to 24